website/docs/docs/build/sl-getting-started.md

```diff
@@ -0,0 +1,118 @@
+---
+id: sl-getting-started
+title: Quickstart
+description: "Learn how to create your first semantic model and metric."
+sidebar_label: Quickstart
+tags: [Metrics, Semantic Layer]
+---
+
+This quickstart guide explains the following steps and recommends a workflow that demonstrates some of MetricFlow's essential features: 
```

Comment:
for this quickstart, do we have a repo we can link to, which has the complete sample data for users?

Reply:
Yup I linked the Jaffle shop repo